solana/program_v2/packages/tests/src/gatekeeper-suite/set-gatekeeper-state.test.ts

```diff
@@ -96,7 +96,13 @@ describe('Gateway v2 Client', () => {
       let gatekeeperAccount = await networkService.getGatekeeperAccount();
       const initialState = gatekeeperAccount?.state;
       // sets the gatekeeper's state to Frozen
-      await networkService.setGatekeeperState(GatekeeperState.Frozen).rpc();
+      await networkService
+        .setGatekeeperState(
+          GatekeeperState.Frozen,
+          undefined,
+          networkAuthority.publicKey
```

Comment:
i don't feel this should be needed to be explicitly set... maybe a problem deeper in the code then?

Reply:
As below